ffi/src/dpapi/mod.rs

```diff
@@ -112,18 +115,19 @@ pub unsafe extern "system" fn DpapiProtectSecret(
             None
         };
 
-        let blob_data = try_execute!(
-            n_crypt_protect_secret(
-                secret.into(),
-                sid,
-                root_key,
-                server,
-                username,
-                password.into(),
-                computer_name
-            ),
-            NTE_INTERNAL_ERROR
-        );
+        // let blob_data = try_execute!(
+        //     n_crypt_protect_secret(
+        //         secret.into(),
+        //         sid,
+        //         root_key,
+        //         server,
+        //         username,
+        //         password.into(),
+        //         computer_name
+        //     ),
+        //     NTE_INTERNAL_ERROR
+        // );
+        let blob_data: Vec<u8> = todo!();
```

Comment:
Now, DPAPI functions are async. But we need to call them synchronously here. We have two options:

* `tokio::spawn_blocking`
* use `maybe_async` crate for `dpapi` crate and generate sync functions.

I didn't try `maybe_async`, but I'm unsure if it will not cause problems with `Transport`.

What do you think?